### ISSUE TITLE
Vulnerable SOLR-Images

### ISSUE THREAD

**Issue by nubam** [docker-solr/docker-solr#399]
We still don't see any updates to docker image versions vulnerable to critical CVE-2021-44228 (log4shell).

In our case we're using v7.5.0:
https://hub.docker.com/layers/solr/library/solr/7.5.0/images/sha256-e3db40fa85e7115d2d1d3eb06f7555b6132e33bd3b6e91b17c0a1690122a7acc?context=explore

When will these versions be patched and updated in your docker repository?

**Comment by baukevdw** [comment]
[7.7.3](https://hub.docker.com/layers/solr/library/solr/7.7.3/images/sha256-ac56c3f14054bea4f43ae6e6ed848544e14e178ef436bf5140227496e90fd34e?context=explore) is updated. I don't think they are gonna patch all those old versions. But you can still build the image yourself.

**Comment by dkd-kaehm** [comment]
Due of #282, they will not be updated.
IMHO the images with tags 7.5.x and 7.6.x should be removed, to prevent the folks to start them at all.

**Comment by adenau** [comment]
How about updating 8.11.0 to 8.11.1 ? Is there any time frame?

**Comment by 3XC1T3D** [comment]
Or 8.9.0 with updated log4j libs due to new findings of the log4j vulnerability

**Comment by olstjos** [comment]
yes @dsmiley  please cut an 8.11.1 release, I just pulled 8.11 and noticed it still has the vulnerable log4j-api-2.13.2.jar and the log4j-core-2.13.2.jar and other log4j jar files

use this command to test 

```
find / -name '*.jar' | grep log4j
```

**Comment by HoustonPutman** [comment]
We are currently limited by issues with the Apache Infrastructure. The new image will hopefully be released by the end of the day.

**Comment by Simulant87** [comment]
is this the PR to follow? https://github.com/docker-library/official-images/pull/11537 I am not sure which repository is responsible.

**Comment by HoustonPutman** [comment]
Yes that is the PR to follow, unfortunately there are a lot of independent steps in the process that we do not have control over. We had to wait for the apache mirrors to have the artifacts available (took around 16 hours), then now we have to wait for the official docker images team to approve the PR to kick off the official docker image build.

The release process starting in Solr 9.0 should be a little more straightforward, but the delay will always be there when we are registered as an 'Official Image'.

**Comment by sylus** [comment]
Can someone confirm Solr is still ok given the below?

The 2nd vulnerability for Log4J on 2.15 was increased in CVE score today after it was discovered that it could be used to perform remote code execution against that version.https://www.lunasec.io/docs/blog/log4j-zero-day-severity-of-cve-2021-45046-increased.

We expect an updated communication from CCCS today but we will need to know if we have any externally facing systems running this version. like 1The second Log4j vulnerability was upgraded from a CVSS score of 3.7x (limited DOS) to a CVSS score of 9.7 (limited RCE), and we explain the impact.

The Log4j version 2.15 CVE score is now 9.0/10 coming up from a 3.7. This would fall in the "Critical" category. We are not recommending taking down any externally facing servers that are running 2.15 YET, but that may change when additional information becomes available. If anyone has a web facing server not running 2.16 it may be a good idea to prepare for the possibility.

**Comment by joejoseph00** [comment]
From the Apache Software Foundation "one of the key principles is Community over code."
Please push a minimal 8.11.1 release even if it means the ugly log files that we saw in the related commit?  I'm suggesting to postpone all other nits for a future release and just get a build out asap.

**Comment by HoustonPutman** [comment]
Once again, we were blocked on the Official Docker images team to approve and merge our PR for the new image. They have just now done this and the new image should be available very soon.

**Comment by HoustonPutman** [comment]
The official Solr image for 8.11.1 is now available on Docker hub: https://hub.docker.com/_/solr

**Comment by olstjos** [comment]
Thank you @HoustonPutman

**Comment by joejoseph00** [comment]
Thanks everyone :1st_place_medal:
